superset-frontend/src/explore/components/DataTableControl/useTableColumns.test.ts

```diff
@@ -28,31 +28,53 @@ const asciiChars = [];
 for (let i = 32; i < 127; i += 1) {
   asciiChars.push(String.fromCharCode(i));
 }
-const asciiKey = asciiChars.join('');
-const unicodeKey = '你好. 吃了吗?';
+const ASCII_KEY = asciiChars.join('');
+const UNICODE_KEY = '你好. 吃了吗?';
+const NUMTIME_KEY = 'numtime';
+const STRTIME_KEY = 'strtime';
+const NUMTIME_VALUE = 1640995200000;
+const NUMTIME_FORMATTED_VALUE = '2022-01-01 00:00:00';
+const STRTIME_VALUE = '2022-01-01';
 
-const data = [
-  { col01: true, col02: false, [asciiKey]: asciiKey, [unicodeKey]: unicodeKey },
-  { col01: true, col02: false, [asciiKey]: asciiKey, [unicodeKey]: unicodeKey },
-  { col01: true, col02: false, [asciiKey]: asciiKey, [unicodeKey]: unicodeKey },
-  {
-    col01: true,
-    col02: false,
-    col03: 'secret',
```

Comment:
The data API never returns "suprise" values for columns that aren't listed in `colnames`, hence we don't need to test for this edge case.